doc/development-context-aware-applications/introduction.md

```diff
@@ -44,12 +44,10 @@ a context-aware application check out:
     - [How to update and query context information](v2/how-to-update-and-query-context-information.md)
     - [How to subscribe to changes on context information](v2/how-to-subscribe-to-changes-on-context-information.md)
     - [How to perform geo-location queries](v2/how-to-perform-geo-located-queries.md)
+    
+  By other hand, you can see a the full API [here](http://fiware.github.io/specifications/ngsiv2/latest/cookbook/)
 
-- Using NGSI v1: 
-    - [How to update and query context information](how-to-update-and-query-context-information.md)
-    - [How to connect to external context providers](how-to-connect-to-external-context-providers.md)
-    - [How to subscribe to changes on context information](how-to-subscribe-to-changes-on-context-information.md)
-    - [How to perform geo-location queries](how-to-perform-geo-located-queries.md)
+It's exist a preview NGSI version, call NGSI v1 but currently is deprecated. Anyway you can check it [here](http://fiware.github.io/context.Orion/api/v1/)
```

Comment:
Not sure if github web is make, but I'm still seing "It exist a preview NGSI version" after b927d9b.

To be more clear: 

* "exist" is incorrect. In English, the plural third person uses "s" at the end, so it must be "exists" just after "it".
* "preview" is incorrect. It's meaning is *"anything that gives an advance impression of something"* and it wouldn't be correct applied to NGSIv1. I guess you actually want to mean "previous" which meanining is *"prior to"*